manager/controllers/utils.go

```diff
@@ -64,9 +64,25 @@ func ApplySort(c *gin.Context, tx *gorm.DB, fieldExprs database.AttrMap,
 	// We sort by a column expression and not the column name. The column expression is retrieved from fieldExprs
 	for _, enteredField := range fields {
 		if strings.HasPrefix(enteredField, "-") && allowedFieldSet[enteredField[1:]] { //nolint:gocritic
-			tx = tx.Order(fmt.Sprintf("%s DESC NULLS LAST", fieldExprs[enteredField[1:]].Query))
+			var orderFields string
```

Comment:
Btw. lines 67-75 and 77-85 are very similar. It would be nice to replace it with the function `createOrderFields(fieldExprs: database.AttrMap, desc: bool) string`.